docs/reference/logging/overview.md

```diff
@@ -172,3 +173,31 @@ The Amazon CloudWatch Logs logging driver supports the following options:
 
 
 For detailed information on working with this logging driver, see [the awslogs logging driver](awslogs.md) reference documentation.
+
+## Specify splunk options
```

Comment:
Also, when creating a separate document, please try to keep a similar structure as the existing documents for consistency.